src/main/webapp/WEB-INF/tags/base.tag

```diff
@@ -22,6 +22,7 @@
 <head>
   <title><%= title != null ? title : "EastCode Chat" %></title>
   <link rel="stylesheet" href="/css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Encode+Sans+Condensed" rel="stylesheet">
```

Comment:
This adds the new font family as a possible font to be used by the app

Reply:
Love this font!